src/lib/journey/config.store.ts

```diff
@@ -0,0 +1,66 @@
+import { readable, type Readable } from 'svelte/store';
+import { z } from 'zod';
+
+export const journeyConfigItemSchema = z.object({
+  journey: z.string().optional(),
+  match: z.string().regex(/^(#\/service|\?journey)/, { message: 'HREF string must start with `?journey` or `#/service`' }).array(),
```

Comment:
This gives us really strong value validation at the runtime, but not perfect on the TypeScript side. I think this is good enough for now, but we can revisit later.

CC @ryanbas21